webui/src/lib/components/navbar.jsx

```diff
@@ -76,7 +77,7 @@ const TopNav = ({logged = true}) => {
         );
     }
     return (
-        <Navbar variant="dark" bg="dark" expand="md">
+        <Navbar variant="dark" bg="dark" expand="md" className="border-bottom">
```

Comment:
@N-o-Z I've added the "border-bottom" for extra separation. It's un-noticable in Light mode, so no real need for extra distinction here.